Mathlib/NumberTheory/Cyclotomic/Rat.lean

```diff
@@ -422,6 +422,80 @@ lemma prime_norm_toInteger_sub_one_of_prime_ne_two' [hcycl : IsCyclotomicExtensi
   replace hζ : IsPrimitiveRoot ζ (p ^ (0 + 1)) := by simpa using hζ
   exact hζ.prime_norm_toInteger_sub_one_of_prime_ne_two hodd
 
+/-- In a `p ^ (k + 1)`-th cyclotomic extension of `ℚ `, we have that `ζ` is not congruent to an
+  integer modulo `p` if `p ^ (k  + 1) ≠ 2`. -/
+theorem not_exists_int_prime_dvd_sub_of_prime_pow_ne_two
```

Comment:
Should this be rather `notExistsIntPrime_dvd_sub_PrimePowNe_two` or something?

Reply:
OK, I agree it should be snake_case, so I take my previous comment back. On the other hand, it seems to me that this lemma is simply an auxiliary one for the next; so I would probably call it `aux` or something, and would specify this in its doc.